README.md

```diff
@@ -1,4 +1,4 @@
-# Mozilla Experimenter 
+# Mozilla Experimenter
```

Comment:
Once [this PR lands](https://github.com/mozilla/experimenter/pull/1027), I'll rebase and these annoying auto-fixes will go away from *this* PR.

Reply:
I think you should be able to do that now.